docs/content/development/extensions-contrib/google.md

```diff
@@ -8,12 +8,15 @@ To use this extension, make sure to [include](../../operations/including-extensi
 
 ## Deep Storage
 
-[Google Cloud Storage](https://cloud.google.com/storage/) is another option for deep storage. This requires some additional druid configuration.
+Deep storage can be written to Google Cloud Storage either via this extension or the [druid-hdfs-storage extension](../extensions-core/hdfs.html).
+
+### Configuration
+
+|Property|Possible Values|Description|Default|
```

Comment:
I think we already have a document for this. Please check http://druid.io/docs/latest/development/extensions-core/hdfs.html#google-cloud-storage.

The removed configurations are actually valid when directly using GCS without Hadoop.

Reply:
We have two extensions that enable using GCS in deep storage: ../extensions-contrib/google.md and ../extensions-core/hdfs.html. When using the google extension, the `druid.storage.type` needs to be `google`, which isn't documented anywhere as far as I know. I was hoping to make it more clear how to use GCS with either this extension or the hdfs extension.

Reply:
Thanks, I got your point. 

`druid.storage.type` should be specified as you said. But, `druid.storage.storageDirectory` is used only in HDFS or local deep storage types. GCS deep storage type needs to set `druid.google.bucket` and `druid.google.prefix` instead.

Reply:
Ah okay. Thanks, Jihoon!